modules/gui/player-list.lua

```diff
@@ -47,7 +78,8 @@ end)
         if player.character then
             player.zoom_to_world(selected_player.position, 1.75)
         else
-            player.teleport(selected_player.position, selected_player.surface)
+            --player.teleport(selected_player.position, selected_player.surface)
```

Comment:
Is there a purpose in leaving the old code commented out?

Reply:
I some times leave in code for old behaviour while testing so I can reference it. Sometimes I forget to remove them after.